packages/core/src/App/Containers/AccountSwitcher/helpers/account-helper.js

```diff
@@ -44,72 +44,3 @@ export const getSortedCFDList = account_list => {
 };
 
 export const isDemo = account => account.account_type === 'demo';
-
```

Comment:
this is a piece of dead code that is found while pairing with BE to inspect if removal of `trading_servers` endpoint will break FE